flocker/cli/script.py

```diff
@@ -108,11 +114,28 @@ def parseArgs(self, control_host, deployment_config, application_config):
                     error=str(e)
                 )
             )
+
+        self["cafile"] = FilePath(self["cafile"])
+        self["cert"] = FilePath(self["cert"])
+        self["key"] = FilePath(self["key"])
+
         if self["certificates-directory"] is None:
             self["certificates-directory"] = FilePath(os.getcwd())
         else:
             self["certificates-directory"] = FilePath(
                 self["certificates-directory"])
+            self["cafile"] = self["certificates-directory"].child(
```

Comment:
:question: I'll test this but it seems like odd behaviour if I specify a certificates directory and a full path to a file in another directory that it'd strip the path apart from the basename.